build/ci/unit-tests.yml

```diff
@@ -61,3 +61,21 @@ jobs:
     pool: 
       name: NetCorePublic-Pool
       queue: BuildPool.Windows.Amd64.VS2019.Pre.ES.Open
+
+- job: RichNav
+  pool:
+    name: NetCorePublic-Pool
+    queue: BuildPool.Windows.10.Amd64.VS2019.Pre.Open
+  variables:
+    _configuration: Debug
+  timeoutInMinutes: 240
```

Comment:
4 hours, is that right?

Reply:
I believe this is correct (it is _very_ slow)

Reply:
I think we'd probably only want to trigger this once a day, or avoid overlapping jobs.

Reply:
(I'm from the rich nav team) Yes, thanks for calling it out. Improving time to build is a critical requirement for us, and we have been working on revamping our internals to improve incrementality, so that indexing does not take this long. We're testing these changes in dev currently. cc @ BertanAygun

Reply:
Last time I looked at the perf for intellinav most of the time was taken up serializing json...